### ISSUE TITLE
<video> elements must have captions

### ISSUE THREAD

**Issue by clever-rob** [GoogleChrome/lighthouse#16244]
<video> elements contain a <track> element with [kind="captions"]

Lighthouse complains about a video element without "captions" track.
I think that this rule is a bit overblown, there are many cases where a captions track cannot and must not be provided.
My case is a QR code scan situation where is no way and reason to provide a captions track.

On a general note, I believe that Lighthouse is "the" tool for web developers, but I wish it does not become 
a tick box exercise caused by all this weird and superfluous rules.

**Comment by clever-rob** [comment]
Yes, it is a QR code scan. There is no way that captions should be applied in this situation.

**Comment by clever-rob** [comment]
Also, the same would apply for any other live video. No captions track in this case either

**Comment by adamraine** [comment]
I forwarded this question to axe-core: https://github.com/dequelabs/axe-core/issues/4638

**Comment by adamraine** [comment]
Axe folks said that videos without captions just need a manual review. So you do not *need* to add captions.

Lighthouse handles this by making the audit informative, which means the audit also doesn't affect the overall accessibility score. There may be ways to clarify this, but otherwise it should be working as intended.
